common-lib/common/panels.libsonnet

```diff
@@ -0,0 +1,6 @@
+local g = import 'g.libsonnet';
+
+{
+  stat: import 'panels/stat/main.libsonnet',
```

Comment:
does this import work? Won't it be looking in the `JSONNET_PATH` for this lib? I'd say `./panels/...` would be better.